src/test/java/stepdefinitions/StepDefinitions.java

```diff
@@ -1,66 +1,116 @@
 package stepdefinitions;
 
-import io.cucumber.java.PendingException;
+import api.RandomApi;
+import entities.Persona;
+import io.cucumber.datatable.DataTable;
 import io.cucumber.java.en.And;
 import io.cucumber.java.en.Given;
 import io.cucumber.java.en.Then;
 import org.openqa.selenium.WebDriver;
-import pages.ContactUsPage;
-import pages.MobileChapterPage;
-import pages.ValoriHomePage;
+import pages.*;
 
 import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
 
+import static org.junit.Assert.assertNotEquals;
 import static org.junit.Assert.assertTrue;
 
 public class StepDefinitions {
-    private WebDriver driver;
+    private final WebDriver driver;
+    private Persona persona;
 
-    public StepDefinitions (){
+    public StepDefinitions() {
         this.driver = DriverManager.driver;
+        this.persona = new Persona();
```

Comment:
Dit is best wel een risico: Je initialiseert een default persona als field. En vervolgens wijzig je die in verschillende methodes. Zou het niet veiliger zijn om de persona aan te maken op de plek waar je hem daadwerkelijk gebruikt zodat je zeker weet dat de data die erin zit op dat moment ook de data is die je verwacht.

Reply:
Je hebt gelijk, maar wilde ook gezien het project niet te complex maken. Als dit een groter project was zouden we moeten kijken naar mooiere oplossingen, misschien iets met context's en/of dependency injection. Dat leek me alleen nu overkill voor de opdracht.

Was er een andere oplossing waar je aan zat te denken?